### PR TITLE
Update asset.ts - fix file path translation from Windows > Linux

### PR DESCRIPTION
**Pull request by Donovoi**
Some of my folders in Windows had special characters like "@" which were not being handled correctly and stopped everything else from being processed. This uses the library "upath" to fix the issue. I'm not a typescript developer so please let me know if I'm doing something incorrect. 

Thank you

### CODE REVIEW

**Comment by JW-CH** [comment]
Can you update your PR? The build fails and you have some strange changes in the imports etc (changed " to ') etc. Please fix the build errors :)

**Comment by Donovoi** [comment]
HI I believe I sorted the issues, but how do I test if the build is ok? I've tried to follow 
https://immich.app/docs/developer/pr-checklist
I also ran make e2e and it seems to run

**Comment by Donovoi** [comment]
Sweet thank you! Let me know when you do and I'll fix up any issues 😁

**Comment by zackpollard** [comment]
You seem to have an extra dependency "install" that is not required, also the builds and tests are failing. Can you fix these two problems please?

**Comment by Donovoi** [comment]
hopefully that's all done!

File: cli/package.json [hunk 1]
```diff
@@ -59,7 +59,9 @@
   },
   "dependencies": {
     "fast-glob": "^3.3.2",
-    "lodash-es": "^4.17.21"
+    "install": "^0.13.0",
```
Comment (zackpollard) [inline cli/package.json:62]: You still have an extra dependency here that has been added erroneously it seems

Comment (Donovoi) [reply cli/package.json]: Awesome thank you for pointing it out. I removed it and re-ran npm install. hopefully its all fixed now 😁

**Comment by bo0tzz** [comment]
I think this has been superseded by #10430?

**Pull request closed without merging**